config/default.yml

```diff
@@ -3065,6 +3065,7 @@ Style/Documentation:
   Description: 'Document classes and non-namespace modules.'
   Enabled: true
   VersionAdded: '0.9'
+  IgnoredConstants: []
```

Comment:
Let's name this `AllowedConstants`. I know we're using `Allowed` and `Ignored` in the existing config, but the more I think about it, the more I like the `allowed` naming, as this is essentially an allowlist.